### PR TITLE
3rd party integrations

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

**Comment by RichardSmedley** [comment]
NOTE

1. This is a wip, for commenting on status of various integrations, and seeing which we can publish to the page in the next few days
(we can then add more later).
2. The source here contains many commented out sections - between //// blocks,
these will not appear in a built page.
3. A better view can be seen at https://github.com/couchbaselabs/docs-devex/blob/3rd-party-integrations/modules/develop/pages/integrations.adoc
4. Or an indication of what the built page will look like:
image
image
image

**Comment by RichardSmedley** [comment]
image

image

image

File: modules/develop/pages/intro.adoc [hunk 1]
```diff
@@ -73,4 +73,11 @@ It enables you to execute code either in response to mutations, or as scheduled
...
-* xref:analytics:introduction.adoc[Introduction to Analytics]
\ No newline at end of file
+* xref:analytics:introduction.adoc[Introduction to Analytics]
+
+== Integrations
+
+Integrations that Couchbase or partners have developed to enhance Couchbase, Capella, and partner services --
+a vital tool for many larger development projects.
+
+* xref:server:develop:integrations.adoc[]
```
Comment (simon-dew) [inline modules/develop/pages/intro.adoc]: You shouldn't need to include the `server` component in the xref coordinates, as this page is already in a "server" branch of `docs-devex` &mdash; the Antora component identifier will ensure that the playbook adds this page in the `server` component.

If you include the `server` component in these coordinates, you'll have to remember to change it to `cloud` when you port this change to the "cloud" branches of `docs-devex`. If you leave out the `server` component, you can port this change to the "cloud" branches without having to change the xref co-ordinates. Generally I try to stop the "cloud" and "server" branches diverging unnecessarily.

(Test the build to make sure the xref still works, of course.)

```suggestion
* xref:develop:integrations.adoc[]
```

File: modules/develop/pages/intro.adoc
Comment (simon-dew) [inline modules/develop/pages/intro.adoc:1]: If you want to keep the number of tiles down (and preserve symmetry 😀) you can get rid of the **Starter Kits** tile:

1.  Starter kits are linked from the left-hand nav.
2. I'm not sure whether the starter kits are maintained.
3. Integrations seem more important than starter kits.

File: modules/develop/pages/integrations.adoc [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+* https://molo17.com/gluesync/[GlueSync homepage]
+| ✔
+| ✔
+| Partner Supported
+
+
+| ODBC / JDBC
+| Easily access live Couchbase data from BI, Analytics, Reporting, ETL, & Custom Apps.
+a|
+* https://docs.couchbase.com/server/current/connectors/odbc-jdbc-drivers.html[Couchbase ODBC and JDBC Drivers]
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc]: Should this be an xref or link? It's in the docs?

File: modules/develop/pages/integrations.adoc [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+* https://github.com/couchbaselabs/couchbase-shell[GitHub repo]
+| ✔
+| ✔ 
+| Couchbase Officially Supported (currently in _Developer Preview_)
+
+
+| cbimport
+| Utilitly for importing data into Couchbase databases.
+Included in self-managed Couchbase Server, or downloadable as part of the xref:cloud:reference:command-line-tools.adoc#download-and-install-the-couchbase-command-line-tools[Capella Tools package].
+a| xref:server:tools:cbimport.adoc[cbimport reference]
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc:138]: Just checking &mdash; is the `cbimport` documentation available in the cloud docs as well? If so, remember to link to the documentation on the cloud side when porting this PR to the cloud branches.

File: modules/develop/pages/integrations.adoc [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+| ✔
+| ✔ 
+| Couchbase Officially Supported
+
+
+| cbexport
+| Utilitly for exporting data from Couchbase databases. 
+Included in self-managed Couchbase Server, or downloadable as part of the xref:cloud:reference:command-line-tools.adoc#download-and-install-the-couchbase-command-line-tools[Capella Tools package].
+a| xref:server:tools:cbexport.adoc[cbexport reference]
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc:147]: Just checking &mdash; is the `cbexport` documentation available in the cloud docs as well? If so, remember to link to the documentation on the cloud side when porting this PR to the cloud branches.

File: modules/develop/pages/integrations.adoc [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+[cols="26,45,30,30,30,28"]
+|===
+| Connector | Summary | Docs | Source | Other Integrations | <<#support-model,Support Model>>
+
+
+| Elasticsearch
+| Replicate your documents from Couchbase Server or Capella to Elasticsearch or OpenSearch in near real time. 
+| xref:elasticsearch-connector:getting-started.adoc[Getting Started docs]
+| https://docs.couchbase.com/elasticsearch-connector/current/release-notes.html[Release notes & download links]
+| https://docs.couchbase.com/elasticsearch-connector/current/autonomous-operations.html[AO Mode]
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc]: Should these be xrefs or links? They're in the docs?

File: modules/develop/pages/integrations.adoc [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+| A plug-in for the Kafka Connect framework, which provides source and sink components.
+| xref:kafka-connector:index.adoc[Kafka connector docs]
+| https://github.com/couchbase/kafka-connect-couchbase[GitHub repo]
+| https://kafka.apache.org/uses[Use cases]
+| Couchbase Officially Supported
+
+
+| Spark
+| Connect to the scalable computing engine for Big Data.
+| https://docs.couchbase.com/spark-connector/current/index.html[Couchbase Spark docs]
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc]: xref or link?

File: modules/develop/pages/integrations.adoc [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+| https://github.com/couchbase/couchbase-spark-connector[Couchbase Spark Connector on GitHub]
+| https://docs.databricks.com/_extras/notebooks/source/couchbase.html[Apache Spark Notebook]
+| Couchbase Officially Supported
+
+
+| Tableau
+| Integrate Couchbase Server tabular views and the Tableau interactive data visualization platform.
+| https://docs.couchbase.com/tableau-connector/current/index.html[Tableau connector docs]
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc]: xref or link?

**Review by simon-dew — COMMENTED** [review]
Added some comments on the content.

As a general question, I agree that integrations should be added to the **Developer's Intro** page. I just want to make sure that `docs-devex` is the best repo to store the **Integrations** page. The `docs-devex` repo is specifically for information which is shared between cloud and server. So if the **Integrations** page is going to be ported to cloud as well, this is the correct place for it. If it's only on server, we need to consider if maybe the `connectors` module on `docs-server` is a better place for it.

File: modules/develop/pages/intro.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/develop/pages/intro.adoc]: This comment above assumes that `integrations.adoc` is going to be in the docs-devex repo of course.

**Comment by simon-dew** [comment]
> So if the **Integrations** page is going to be ported to cloud as well, this is the correct place for it.

Okay, looking at #84, it seems that the Integrations page _should_ be ported to the cloud branch as well. 😀

File: modules/develop/pages/integrations.adoc (hunk 3 above)
Comment (RichardSmedley) [reply modules/develop/pages/integrations.adoc:138]: Do you think the link in the previous cell is enough?

File: modules/develop/pages/integrations.adoc (hunk 4 above)
Comment (RichardSmedley) [reply modules/develop/pages/integrations.adoc:147]: Do you think the link in the previous cell is enough?

File: modules/develop/pages/integrations.adoc (hunk 5 above)
Comment (RichardSmedley) [reply modules/develop/pages/integrations.adoc]: :+1:

File: modules/develop/pages/integrations.adoc (hunk 2 above)
Comment (RichardSmedley) [reply modules/develop/pages/integrations.adoc]: :+1:

File: modules/develop/pages/integrations.adoc (hunk 5 above)
Comment (RichardSmedley) [reply modules/develop/pages/integrations.adoc]: :+1:

File: modules/develop/pages/integrations.adoc (hunk 7 above)
Comment (RichardSmedley) [reply modules/develop/pages/integrations.adoc]: :+1:

File: modules/develop/pages/intro.adoc
Comment (RichardSmedley) [reply modules/develop/pages/intro.adoc:1]: Will do - although updating the starter kits page is still on my todo list  :sweat_smile:

**Comment by RichardSmedley** [comment]
> As a general question, I agree that integrations should be added to the **Developer's Intro** page. I just want to make sure that `docs-devex` is the best repo to store the **Integrations** page. The `docs-devex` repo is specifically for information which is shared between cloud and server. So if the **Integrations** page is going to be ported to cloud as well, this is the correct place for it. If it's only on server, we need to consider if maybe the `connectors` module on `docs-server` is a better place for it.

Connectors is only a subset of what's on offer....

I take your point about cross server/cloud,
however atm the home of this page is unsettled, and it's going out today to time with cloud-related stuff,
so this seems the best home for now.

Happy to quickly move it and add a redirect at any time - including the next 30 mins if necessary  ;)

**Comment by RichardSmedley** [comment]
> > So if the **Integrations** page is going to be ported to cloud as well, this is the correct place for it.
> 
> Okay, looking at #84, it seems that the Integrations page _should_ be ported to the cloud branch as well. grinning

Again - maybe, things are in flux.

File: modules/develop/pages/integrations.adoc (hunk 3 above)
Comment (simon-dew) [reply modules/develop/pages/integrations.adoc:138]: Fine for now!

File: modules/develop/pages/integrations.adoc (hunk 4 above)
Comment (simon-dew) [reply modules/develop/pages/integrations.adoc:147]: Fine for now!

File: modules/develop/pages/integrations.adoc (hunk 7 above)
Comment (simon-dew) [reply modules/develop/pages/integrations.adoc]: Think this one's been missed?

```suggestion
| xref:tableau-connector::index.adoc[Tableau connector docs]
```

File: modules/develop/pages/integrations.adoc [hunk 8]
```diff
@@ -0,0 +1,343 @@
...
+a| 
+* https://couchbase.sh/docs/[cbsh docs]
+* https://github.com/couchbaselabs/couchbase-shell[GitHub repo]
+| ✔
+| ✔ 
+| Couchbase Officially Supported (currently in _Developer Preview_)
+
+
+| cbimport
+| Utilitly for importing data into Couchbase databases.
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc]: Typo

```suggestion
| Utility for importing data into Couchbase databases.
```

File: modules/develop/pages/integrations.adoc [hunk 9]
```diff
@@ -0,0 +1,343 @@
...
+Included in self-managed Couchbase Server, or downloadable as part of the xref:cloud:reference:command-line-tools.adoc#download-and-install-the-couchbase-command-line-tools[Capella Tools package].
+a| xref:server:tools:cbimport.adoc[cbimport reference]
+| ✔
+| ✔ 
+| Couchbase Officially Supported
+
+
+| cbexport
+| Utilitly for exporting data from Couchbase databases.
```
Comment (simon-dew) [inline modules/develop/pages/integrations.adoc]: Typo

```suggestion
| Utility for exporting data from Couchbase databases. 
```

**Review by simon-dew — CHANGES_REQUESTED** [review]
Second round of review

**Review by simon-dew — APPROVED** [review]

**Pull request merged**